app/views/employees/index.js.erb

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Use a ajax success listener.
http://stackoverflow.com/questions/9154874/ajax-in-rails-show-modelnew-form-after-completing-a-modelupdate/9156082#9156082